packages/api/src/external/carequality/shared.ts

```diff
@@ -168,23 +174,3 @@ export async function getCqAddress({
 export const cqOrgActiveSchema = z.object({
   active: z.boolean(),
 });
-
-export async function getParsedCqOrgOrFail(
```

Comment:
Refactored to [getCqOrgOrFail](https://github.com/metriport/metriport/pull/3131/files#diff-9c0919af4b14ca461759ae7a828713edf4f1a3bfa70c6a480b6b5443a02e9039R32)